src/System.Private.CoreLib/src/System/Runtime/CompilerServices/RuntimeHelpers.cs

```diff
@@ -157,6 +157,7 @@ private static unsafe int MakeHashCode(Object o, int* pSyncBlockIndex)
 
         public static int OffsetToStringData
         {
+            [Intrinsic] // Workaround to allow WebAssembly to define a size here without a special CoreLib build
```

Comment:
Could you add a link to the issue that tracks removing this?

Reply:
Done